test/tree-sitter-corpus.jl.faceup

```diff
@@ -0,0 +1,784 @@
+«x:# ==============================»
+«x:# tuple collections»
+«x:# ==============================»
+
+()
+«x:# There's no (,)»
+
+(«c:1») «x:# NOT a tuple»
+(«c:1»,)
+(«c:2»,«c:3»,«c:4»,)
+
+«x:# ==============================»
+«x:# named tuple collections»
+«x:# ==============================»
+
+(«v:a» = «c:1») «x:# NOT a tuple»
```

Comment:
This shouldn't be `font-lock-variable-name-face`.

Reply:
Wait. I missed the whole point of this test case. Sorry.

Reply:
As a matter of fact, it should. As the comment correctly states, this is not a tuple, but a variable assignment, in parentheses. `a` is correctly identified as a variable, and highlighted as such.